include/swift/AST/Module.h

```diff
@@ -888,7 +888,8 @@ class SourceFile final : public FileUnit {
   ASTStage_t ASTStage = Parsing;
 
   SourceFile(ModuleDecl &M, SourceFileKind K, Optional<unsigned> bufferID,
-             ImplicitModuleImportKind ModImpKind, bool KeepSyntaxInfo);
+             ImplicitModuleImportKind ModImpKind, bool KeepTokenSyntax = false,
```

Comment:
`TokenRecorder` is not related to `libSyntax`. I feel `KeepParsedTokens` or such are more relevant.